ftw/keywordwidget/field.py

```diff
@@ -33,7 +29,7 @@ def _validate(self, value):
                 raise ValueError("Can't validate value without vocabulary")
 
         # The widget can control this attribute too
-        if self.allow_new:
+        if getattr(api.portal.getRequest(), 'allow_new', True):
```

Comment:
I would use `False` as the default value. Otherwise it would be possible in some edge-cases to succeed the validation without permission.

Reply:
I'm under the impression one would use the `Choice` field where ever one would want that and this is used explicitly to allow extending (unless explicitly blacklisted).

I'm not aware of how this is actually used outside `opengever.core`, so this should apparently then be evaluated company wide.

Reply:
We cannot guarantee that the `ChoicePlus` is validated after updating the widget. We actually have use cases where there is no widget involved at all, e.g. restapi, I think.
That means: if we default to `False` it will be impossible to add new terms with those methods which have no widget.

Therefore we should default to `True`.